sjb/config/test_cases/test_pull_requests_s2i.yml

```diff
@@ -0,0 +1,32 @@
+---
+parent: 'common/test_cases/minimal.yml'
+extensions:
+  actions:
+  - type: "script"
+    title: "build and install s2i"
+    timeout: 300
+    repository: "source-to-image"
+    script: |-
+      export BUILD_GO_VERSION=go1.12.5 
+      wget -q https://dl.google.com/go/${BUILD_GO_VERSION}.linux-amd64.tar.gz && sudo tar -C /usr/local -xzf ${BUILD_GO_VERSION}.linux-amd64.tar.gz && sudo rm /bin/go* && sudo ln -s /usr/local/go/bin/go /usr/local/go/bin/godoc /usr/local/go/bin/gofmt /bin
+      make build
+      sudo cp /data/src/github.com/openshift/source-to-image/_output/local/bin/linux/amd64/s2i /usr/bin
+      echo "Printing s2i version"
+      s2i version
+  - type: script
+    title: build test images
+    repository: source-to-image
+    script: |-
+      hack/build-test-images.sh
```

Comment:
Turns out I didn't need the pull secret to registry.redhat.io after all (though having it available on ci.openshift.redhat.com isn't a bad thing - someone will use the UBI for their images sooner or later)

Reply:
cool deal .... and refresh my memory, with https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/source-to-image/openshift-source-to-image-master-presubmits.yaml#L132-L141 you were able to confirm recently that running both prow based and jenkins based tests out of the same PR was functional, right?

In theory I could employ a similar pattern with https://github.com/openshift/aos-cd-jobs/blob/master/sjb/config/test_cases/test_branch_jenkins_images.yml to bring back those old docker/s2i based ginkgo tests

Reply:
> and refresh my memory, with https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/source-to-image/openshift-source-to-image-master-presubmits.yaml#L132-L141 you were able to confirm recently that running both prow based and jenkins based tests out of the same PR was functional, right?

Yes. Caveat here is that you need to hand-edit the `jobs/*.yaml` file - the config generator won't let you add `jenkins` agent tests.